matching-web/src/main/java/com/graphhopper/matching/http/MatchResultToJson.java

```diff
@@ -75,9 +75,9 @@ public JSONObject exportTo() {
 
             for (GPXExtension extension : edgeMatch.getGpxExtensions()) {
                 JSONObject wpt = new JSONObject();
-                wpt.put("x", extension.getQueryResult().getSnappedPoint().lon);
-                wpt.put("y", extension.getQueryResult().getSnappedPoint().lat);
-                wpt.put("timestamp", extension.getEntry().getTime());
+                wpt.put("x", extension.queryResult.getSnappedPoint().lon);
```

Comment:
I'm usually a friend of public final stuff except where we guess that we could use subclasses and method overwriting, but the problem here is that this breaks API compatiblity for no (good) reason IMO